client/app/bundles/course/group/pages/GroupShow/GroupManager/GroupUserManagerList.jsx

```diff
@@ -81,13 +90,49 @@ const styles = {
   },
 };
 
+const GroupUserManagerListItemChoice = ({
+  isOutsideGroup,
+  user,
+  onChangeDropdown,
+}) => {
+  if (isOutsideGroup || !user.groupRole) return undefined;
+  return user.role === 'student' ? (
```

Comment:
Do we need this role check? They are not supposed to be able to enter this page no?